woocommerce/woocommerce
# Updated `wp-env` To 8.0.0

### Submission Review Guidelines:

- I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/). 
- Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

We are updating `@wordpress/env` to 8.0.0 in order to consume some upstream fixes.

- `wp-env` no longer requires WSL to function.
- Fixes to `wp-env run` that no longer require us to use quotes (and in fact, break, when we do)



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Run `pnpm --filter=woocommerce env:test` after updating.
2. Run `pnpm --filter=woocommerce test:unit:env` and confirm they pass.
3. Run `pnpm --filter=woocommerce test:e2e-pw` and confirm they pass.
